pkg/ocm/helpers.go

```diff
@@ -354,9 +356,19 @@ func (c *Client) LinkOrgToRole(orgID string, roleARN string) (bool, error) {
 	return true, nil
 }
 
+func (c *Client) IsRoleLinked(orgID string, roleARN string) (bool, error) {
+	resp, err := c.ocm.AccountsMgmt().V1().Organizations().Organization(orgID).
+		Labels().Labels(STSOCMRole).Get().Send()
+	if err != nil {
+		return false, err
+	}
+
+	return roleARN == resp.Body().Value(), nil
```

Comment:
Can we assume that the `sts_ocm_role` label value contains only a single roleARN?

Reply:
No you have to iterate through it and check for the aws account id and then list it

Reply:
I've noticed that the **roleARN** and the **sts-ocm-role** label both contains the AWS account id,
for example: "arn:aws:iam::**765374464689**:role/tavni26a-OCM-Role-12541229.
I've changed this line, is it sufficient? 
If we have multiple values in this label separated with commas, I think this check: 
`strings.Contains(resp.Body().Value(), roleARN)` will work. 
WDYT?